Criterion/Main/Options.hs

```diff
@@ -27,8 +27,7 @@ import Data.Semigroup ((<>))
 
 import Control.Monad (when)
 import Criterion.Analysis (validateAccessors)
-import Criterion.Types (Config(..), Verbosity(..), measureAccessors,
-                        measureKeys)
+import Criterion.Types (Config(..), Verbosity(..), measureAccessors, measureKeys)       
```

Comment:
In general, I'd prefer not to clutter up the diff with minor aesthetic changes like this unless there's a good reason for them (which there doesn't seem to be here).